setup.py

```diff
@@ -414,6 +415,37 @@ def has_html(self):
     sub_commands.append(('install_html', has_html))
 
 
+class CheckTestsErrors(Command):
+    """ Checks tests summary.json for failed tests and raises errror if found """
+    description = "Checks tests summary.json for failed tests and raises errror if found"
+    
+    user_options = [
+        ("summary-file=",None,"Path to summary.json"),
+        ("ignore-gevent-errors",None,"Failed tests using GreenMode.Gevent will not raise error."),
```

Comment:
Please run a linter over you new changes - should be spaces after commas in the tuples, for PEP8.  Also in the lists and function argument lists below.